crm_rma_lot_mass_return/wizards/returned_lines_from_serial.py

```diff
@@ -0,0 +1,343 @@
+# -*- coding: utf-8 -*-
+#########################################################################
+#    Module Writen to OpenERP, Open Source Management Solution
+#    Copyright (C) OpenERP Venezuela (<http://www.vauxoo.com>).
+#    All Rights Reserved
+# ############ Credits ##################################################
+#    Coded by: Yanina Aular <yani@vauxoo.com>
+#    Planified by: Yanina Aular <yani@vauxoo.com>
+#    Audited by: Nhomar Hernandez <nhomar@vauxoo.com>
+#########################################################################
+#                                                                       #
+# Copyright (C) 2009-2011  Akretion, Emmanuel Samyn                     #
+#                                                                       #
+# This program is free software: you can redistribute it and/or modify  #
+# it under the terms of the GNU General Public License as published by  #
+# the Free Software Foundation, either version 3 of the License, or     #
+# (at your option) any later version.                                   #
+#                                                                       #
+# This program is distributed in the hope that it will be useful,       #
+# but WITHOUT ANY WARRANTY; without even the implied warranty of        #
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         #
+# GNU General Public License for more details.                          #
+#                                                                       #
+# You should have received a copy of the GNU General Public License     #
+# along with this program.  If not, see <http://www.gnu.org/licenses/>. #
+#########################################################################
+
+from openerp import models, fields, api, _
+from openerp.exceptions import except_orm
+
+
+class ReturnedLinesFromSerial(models.TransientModel):
+
+    _name = 'returned_lines_from_serial.wizard'
+    _description = 'Wizard to create product return lines from serial numbers'
+
+    # Get partner from case is set to filter serials
+    @api.model
+    def _get_default_partner_id(self):
```

Comment:
Why this method is two different types for the same variable it is incorrect.

It should say:

return partner_id

without the if, if browse is False it can be managed already.